### ISSUE TITLE
UI stackable menu does not stack the child "right menu"

### ISSUE THREAD

**Issue by mixerp** [Semantic-Org/Semantic-UI#3604]
The `right menu` inside `ui stackable menu` does not display correctly on mobile devices

``` html
<div class="ui stackable menu">
  <a class="item">Browse</a>
  <a class="item">Submit</a>
  <div class="right menu">
    <a class="item">Sign Up</a>
    <a class="item">Help</a>
  </div>
</div>
```

Example:
http://jsfiddle.net/7wpm7bka/

**Comment by ThreeSevenths** [comment]
I've been struggling with this as well.

**Comment by jlukic** [comment]
This should be easy

**Comment by eveevans** [comment]
Do you want to handle this ? 
or need help ? 
:)

**Comment by jlukic** [comment]
Got it

**Comment by mixerp** [comment]
Thanks jack. You're a rockstar!

**Comment by eskiesirius** [comment]
I am still having this kind of problem, how to fix this?

**Comment by Unnumbered** [comment]
@eskiesirius you may use '.right.item' instead of '.right.menu .item'.

**Comment by ronlut** [comment]
I'm still getting this behavior as well..
Any plans to fix it? @jlukic

**Comment by cidadao** [comment]
+1: Only the first item on the right menu appears stacked along with the left menu items

**Comment by lednhatkhanh** [comment]
+1 This need to be fixed!

**Comment by joexanderson** [comment]
How can we fix this? It's still an issue as only the first item stacks. Others don't.

**Comment by arlyon** [comment]
here's an illustration of the problem, and a (bad) fix to illustrate what is happening to the other items:

https://jsfiddle.net/arlyon/h5e877mm/

Essentially the other items in `.menu.right` are still being placed horizontally. The parent menu correctly stacks as it should, but `.menu.right.stackable` doesn't stack the items in menu right to match its parent.

**Comment by wtfuii** [comment]
This solved it for me:

```html
<div class="ui stackable menu">
    <a class="item">My first left item</div> 
    <a class="item">My second left item</a>
    <a class="right item">My first right item</a> 
    <a class="item">My second right item</a> 
</div>
```

There may be a missing border on the right side, but currently this seems to be the best solution.

**Comment by droid-sheep** [comment]
It's rightthe solution from @wtfuii has worked perfectly for me. But you have to deal with the margin-left of the .items

**Comment by BleuDiamant** [comment]
Hello,
Problem alway present at this date ;)

**Comment by Traverse** [comment]
I think the following addition would be sufficient to fix this issue, it does require some testing though.

in `menu.less`:
```less
@media only screen and (max-width: @largestMobileScreen) {

  ...

  .ui.stackable.menu .right.menu,
  .ui.stackable.menu .left.menu {
    flex-direction: column;
  }
}
```
If someone else would like to create a pull request go ahead and do it since I won't have enough time at the moment to do it.

**Comment by jlukic** [comment]
I've fixed this in `2.2.11`

**Comment by imtinge** [comment]
when will the 2.2.11 release?
